corehq/apps/app_manager/models.py

```diff
@@ -806,8 +807,10 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            form = self.wrapped_xform()
+            form.strip_vellum_ns_attributes()
             try:
-                validate_xform(self.source,
+                validate_xform(etree.tostring(form.xml),
```

Comment:
This is needed because formtranslate required each attribute to be a valid xpath and hashtags are definitely not.